lexos/processors/analyze/contentanalysis_receiver.py

```diff
@@ -0,0 +1,47 @@
+from lexos.receivers.base_receiver import BaseReceiver
+
+
+class ContentAnalysisOption:
+    def __init__(self, formula, dict_label, toggle_all, dict_labels,
+                 active_dicts, toggle_all_value):
+        self.formula = formula
+        self.dict_label = dict_label
+        self.toggle_all = toggle_all
+        self.dict_labels = dict_labels
+        self.active_dicts = active_dicts
+        self.toggle_all_value = toggle_all_value
+
+
+class ContentAnalysisReceiver(BaseReceiver):
+
+    def __init__(self):
+        """The Receiver to get all the content analysis options"""
+        super().__init__()
+
+    def options_from_front_end(self) -> ContentAnalysisOption:
+        """Get the content analysis option from front end
+
+        :return: a ContentAnalysisOption object to hold all the options
+        """
+        options = [Option('calc_input', None),
```

Comment:
There is truly no point in doing this dude. Also this is really insecure, every single option can possibly be `None`? Then you need to actually handle the `None` case in your code. 
This is not type-safe, and I think this code will not be able to pass mypy's check with strict mode on.